test/functions.test.js

```diff
@@ -40,53 +93,1521 @@ test('greetUsersOverAge60', (expect) => {
     expect.deepEqual(actual, expected);
 });
 
-skip('addAllAges', (expect) => {
-    const expected = true;
-
-    const actual = true;
+test('addAllAges', (expect) => {
+    const expected = 2125;
 
+    const actual = addAllAges(customers);
     expect.equal(actual, expected);
 });
 
 
-skip('getAverageCoolFactor', (expect) => {
-    const expected = true;
+test('getAverageCoolFactor', (expect) => {
+    const expected = 5.088888888888889;
 
-    const actual = true;
+    const actual = getAverageCoolFactor(customers);
 
     expect.equal(actual, expected);
 });
 
-skip('getTotalOfEachGender', (expect) => {
-    const expected = true;
+test('getTotalOfEachGender', (expect) => {
+    const expected = {
+        'Bigender': 1,
+        'Female': 19,
+        'Genderqueer': 1,
+        'Male': 23,
+        'Non-binary': 1
+    };
 
-    const actual = true;
+    const actual = getTotalOfEachGender(customers);
 
-    expect.equal(actual, expected);
+    expect.deepEqual(actual, expected);
 });
 
-skip('getGenderBreakdownOfFordOwners', (expect) => {
-    const expected = true;
+test('getGenderBreakdownOfFordOwners', (expect) => {
+    const expected = {
+        'Female': 1,
+        'Male': 2
+    };
 
-    const actual = true;
+    const actual = getGenderBreakdownOfFordOwners(customers);
 
-    expect.equal(actual, expected);
+    expect.deepEqual(actual, expected);
 });
 
-skip('getGenderBreakdownOfEachCar', (expect) => {
-    const expected = true;
+test('getGenderBreakdownOfEachCar', (expect) => {
+    const expected = {
+        'Alfa Romeo': {
+            'Bigender': 1,
+            'Female': 19,
+            'Genderqueer': 1,
+            'Male': 23,
+            'Non-binary': 1
+        },
```

Comment:
Really close on this, notice how these are the same numbers as our TOTAL gender breakdown. I would presume 23 males don't own alfa romeo's and audi's and bmw's. You just need to tweak something inside your reduce that will just look at each car type